src/scikit_build_core/build/wheel.py

```diff
@@ -30,6 +30,7 @@
 
 if TYPE_CHECKING:
     from collections.abc import Sequence
+    from typing import Iterable
```

Comment:
```suggestion
    from collections.abc import Iterable, Sequence
```

The typing imports are deprecated.